README.md

```diff
@@ -5,11 +5,12 @@ A browser extension that gives you insight into your involvement with [Web Monet
 
 Akita presents your top visited monetized sites, how much time you’re spending on them, and how much you're contributing (or could contribute) to them.
 
-We've built this for the [Grant for the Web x DEV hackathon](https://res.cloudinary.com/practicaldev/image/fetch/s--aIp9TotN--/c_imagga_scale,f_auto,fl_progressive,h_500,q_auto,w_1000/https://dev-to-uploads.s3.amazonaws.com/i/q19vviykh0oi1s5tkbqe.png)!
+We've built this for the [Grant for the Web x DEV hackathon](https://dev.to/devteam/announcing-the-grant-for-the-web-hackathon-on-dev-3kd1)! We're so thrilled to have received a Runner Up Award for this project 🎉 - [Winners Announcement](https://dev.to/devteam/announcing-the-grant-for-the-web-x-dev-hackathon-winners-1nl4).
 
 ---
 
 ## Table of Contents
+  - [Licenses](#licenses)
```

Comment:
Although licensing is certainly important, I'm not sure it'll be the first thing people are interested in knowing about the project. Perhaps we should move this section to later in the README.

Reply:
I agree it's not the most exciting thing, but awareness of/understanding the licenses is a bit of a pre-req to installing/contributing to Akita.

We could alternatively add a file in the docs folder with these license details, and just include the MIT/CC shields below the Akita heading with a note to refer to the license doc. So it's clear we are using these licenses and they can read more if they wish. Maybe we'd want to link to the license doc in our installing doc as well, in that case.

Reply:
True awareness and understanding of the licenses are important (we experienced it ourselves with Flickity lol).

I like that idea. Solves the problem of taking up the start of the README and I think it will be even more readable/noticeable than text.